docs/reference/run.md

```diff
@@ -38,6 +39,10 @@ The second difference is the `docker-compose run` command does not create any of
 
     $ docker-compose run --service-ports web python manage.py shell
 
+Alternatively manual port mapping can be specified. Same as when running Docker's `run` command - using `--publish` or `-p` options:
+
+    $ docker-compose run --publish 8080:80 -p 2022:22 web python manage.py shell
```

Comment:
Should this also support the `ip:hostPort:containerPort` form (i.e. allow specifying an IP-address)? (t.b.h., I never needed it, but others may)